### PR TITLE
Upgrade github.com/aws/aws-sdk-go from v1.32.1 to v1.34.1

### PR DESCRIPTION
**Pull request by rinckm**
Upgrade github.com/aws/aws-sdk-go from v1.32.1 to v1.34.1

### Dependency release notes

Upgrade github.com/aws/aws-sdk-go from v1.32.1 to v1.34.1

### Submitter checklist

- [x] Change has been tested (on a back-end cluster)
  - stewardci/stewardci-run-controller:TEST_201111_d302c7e tested OK
- [x] [changelog.yaml] with upgrade notes are prepared and appropriate for the audience affected by the change (users or developer, depending on the change).
- [x] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)

In case dependencies have been updated:
- [x] Links to external changelogs added since the last release of our component
- [ ] Changelogs read thoroughly, potential impact described, upgrade notes prepared (if necessary)
- [x] Check if dependency update affects our semantic version increment.

### Reviewer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There is at least one approval for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] The Pull Request title is understandable and reflects the changes well
- [x] The Pull Request description is understandable and well documented
- [x] 'Upgrade notes' are documented in changelog.yaml (if required)
- [x] [changelog.yaml] entry for this Pull Request has been added
  - [x] Changelog entry contains all required information

[changelog.yaml]: https://github.com/SAP/stewardci-core/changelog.yaml
[releases]: https://github.com/SAP/stewardci-core/releases

### CODE REVIEW

File: changelog.yaml [hunk 1]
```diff
@@ -21,6 +21,12 @@
 - version: "NEXT"
   date: TBD
   changes:
+  - title: Upgrade github.com/aws/aws-sdk-go
```
Comment (anbrsap) [inline changelog.yaml:24]: For dep updates we don't need to be specific in the title:
```suggestion
  - title: Upgrade Go dependencies
```

File: changelog.yaml [hunk 2]
```diff
@@ -21,6 +21,12 @@
...
+    type: internal
+    impact: patch
+    description: |-
+      - upgrade github.com/aws/aws-sdk-go from v1.32.1 to v1.34.1
+        (see [changelog](https://github.com/aws/aws-sdk-go/releases/tag/v1.34.1))
```
Comment (anbrsap) [inline changelog.yaml:29]:
```suggestion
      - upgrade `github.com/aws/aws-sdk-go` from v1.32.1 to v1.34.1
        (see [changelog](https://github.com/aws/aws-sdk-go/releases/tag/v1.34.1))
```

**Review by anbrsap — APPROVED** [review]
You may add the CVEs solved by the update to the PR description and change log.

File: changelog.yaml [hunk 3]
```diff
@@ -21,6 +21,12 @@
 - version: "NEXT"
   date: TBD
   changes:
+  - title: Upgrade Go dependencies
+    type: internal
+    impact: patch
+    description: |-
+      - upgrade `github.com/aws/aws-sdk-go` from v1.32.1 to v1.34.1
+        (see [changelog](https://github.com/aws/aws-sdk-go/releases/tag/v1.34.1))
```
Comment (alxsap) [inline changelog.yaml:30]: `pullRequestNumber` missing

**Pull request merged**